.circleci/config.yml

```diff
@@ -451,50 +427,100 @@ jobs:
           at: .
       - run:
           name: Running E2E Tests
-          command: yarn task --task e2e-tests --template $(yarn get-template << parameters.cadence >> e2e-tests) --no-link --start-from=never --junit
+          command: yarn task --task e2e-tests --template $(yarn get-template << pipeline.parameters.workflow >> e2e-tests) --no-link --start-from=never --junit
       - store_test_results:
           path: test-results
       - store_artifacts: # this is where playwright puts more complex stuff
           path: code/playwright-results/
           destination: playwright
 
 workflows:
-  daily-tests:
+  ci:
     when:
-      equal: [ daily-tests, << pipeline.parameters.workflow >> ]
+      and:
+        - equal: [ api, << pipeline.trigger_source >> ]
+        - equal:  [ ci, << pipeline.parameters.workflow >> ]
     jobs:
       - build
+      - lint:
+          requires:
+            - build
+      - check:
+          requires:
+            - build
+      - unit-tests:
+          requires:
+            - build
+      - script-unit-tests:
+          requires:
+            - build
       - create-sandboxes:
-          parallelism: 24
-          cadence: "daily"
           requires:
             - build
-      # - smoke-test-sandboxes: # disabled for now
```

Comment:
we might want to create an issue to resolve this (reenable when whatever blocker is blocking this gets resolved), else we might forget entirely?

Reply:
Good idea, added an linear ticket in our project